docs/tutorials/services/webcam-line-follower-robot.md

```diff
@@ -85,103 +170,96 @@ Or if you prefer the raw JSON:
       },
       "depends_on": []
     }
+  ]
+}
 ```
 
-### Configuring a color detector for the color of your tape line
+Click **Save config** in the bottom left corner of the screen.
 
-We’ll use the Viam [Vision Service color detector](/services/vision/detection/) to identify the line to follow.
+{{% /tab %}}
+{{< /tabs >}}
 
-- Use a color picker like [colorpicker.me](https://colorpicker.me/) to approximate the color of your line and get the corresponding hexadecimal hash to put in your config.
-  Put this hash in the `detect_color` parameter.
-  We used #19FFD9 to represent the color of green electrical tape.
-- We used a segment size of 100 pixels, and a tolerance of 0.06, but you can tweak these later to fine tune your line follower.
+## Test your components
 
-{{< tabs >}}
-{{% tab name="Builder" %}}
+Navigate to your [robot's Control tab](/manage/fleet/robots/#control) to test your components.
+Verify that it’s connected by refreshing the page and ensuring that **Last Online** (in the top banner) says, "Live."
 
-Navigate to the [robot page on the Viam app](https://app.viam.com/robots).
-Click on the robot you wish to add the Vision Service to.
-Select the **Config** tab, and click on **Services**.
+1. Go to the **Control** tab, click on the base panel, and toggle the transform camera to on.
+   Ensure the camera works as expected.
 
-Scroll to the **Create Service** section.
-To create a [Vision Service](/services/vision/):
+1. Enable the keyboard controls and move the base using your keyboard.
+   Ensure that the base moves as expected.
 
-1. Select `vision` as the **Type**.
-2. Enter `green_detector` as the **Name**.
-3. Select `color_detector` as the **Model**.
-4. Click **Create Service**.
+   {{< alert title="Tip" color="tip" >}}
+   If one or both drive motors are going backwards, you can power down the Pi by running `sudo poweroff`, unplug the battery, and switch the wires to the motor before powering it back on.
+   {{< /alert >}}
```

Comment:
^ not my writing - haven't actually even set this up ;). You'd have to ask the Original author ;)

Reply:
This happens to people in general with all rovers, because with a plain DC motor there is no inherent forwards or backwards way to wire it. So there's always a 50/50 chance of having it backwards :D

Reply:
Ah, thanks!! I was wondering -- very helpful!